### PR TITLE
Fixed minor issue with D.O. maps, refs #10330

### PR DESCRIPTION
**Pull request by mcantelon**
Corrected an incorrect link provided on the default page elements settings
page. Allowed setting to be set to nothing (empty value).

### CODE REVIEW

File: apps/qubit/modules/settings/templates/pageElementsSuccess.php [hunk 1]
```diff
@@ -54,7 +54,7 @@
               <?php echo $form->toggleDigitalObjectMap->label('Digital object map')->renderLabel() ?>
               <?php if (!$googleMapsApiKeySet): ?>
                 <div class="description">
-                  <?php $securitySettingsLinkHtml = link_to(__('Security'), array('module' => 'settings', 'action' => 'security')) ?>
+                  <?php $securitySettingsLinkHtml = link_to(__('global'), array('module' => 'settings', 'action' => 'global')) ?>
                   <?php echo __('This feature will not work until a Google Maps API key is specified on the %1% settings page.', array('%1%' => $securitySettingsLinkHtml)) ?>
```
Comment (sevein) [inline apps/qubit/modules/settings/templates/pageElementsSuccess.php]: There is a way to merge these two i18n strings as one. You have one example here: https://github.com/artefactual/atom/blob/1d80f3826fcbdf318a8c99fa37fc2d33da42fbcd/plugins/sfIsdiahPlugin/modules/sfIsdiahPlugin/actions/indexAction.class.php#L71

Comment (mcantelon) [reply apps/qubit/modules/settings/templates/pageElementsSuccess.php]: Cool... will go that route.

Comment (mcantelon) [reply apps/qubit/modules/settings/templates/pageElementsSuccess.php]: I fixed it but change isn't being reflected by GitHub in the over snippet (prolly as I rebased).

**Comment by sevein** [comment]
LGTM!

**Comment by mcantelon** [comment]
Awesome... thanks!

**Pull request closed without merging**